src/test/java/hw6_jdi/JdiTest.java

```diff
@@ -0,0 +1,94 @@
+package hw6_jdi;
+
+import com.epam.jdi.light.driver.WebDriverUtils;
+import com.epam.jdi.light.elements.init.PageFactory;
+import hw6_jdi.entities.MetalsColors;
+import hw6_jdi.entities.MetalsColorsJ;
+import hw6_jdi.entities.User;
+import hw6_jdi.forms.MetalsColorsForm;
+import hw6_jdi.jdiPages.LeftMenuSection;
+import org.testng.Assert;
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static hw6_jdi.JdiSite.*;
+
+public class JdiTest {
+
+    @BeforeSuite(alwaysRun = true)
+    public void setUp() {
+        PageFactory.initElements(JdiSite.class);
+    }
+
+    @Test(dataProviderClass = JdiDataProvider.class, dataProvider = "provideDataSetForMetalColortest")
+    public void MetalsAndColorsPageTest(MetalsColorsJ dataSet) {
```

Comment:
Oh my, this is quite hard to realise the business value of this particular test.
In general, you should not write such code as line 35, because nobody can understand it at once.
I don't really understand the main idea of 2 similar classes(MetalsColors and MetalsColorsJ) but if you certanly need it you should create Decorator of smt like that. In the other words, IN TEST body you should not create the instance of the class by using constructor with lots of arguments.

Reply:
One of two similar class was deleted